File: PHP/uberVan.php
<?php
require_once('car.php');
require_once 'drver.php';
class UberVan extends Car {
    public $typeCarAccepted;
    public $seatsMaterial;

    public function __construct($license, Drver $driver, $typeCarAccepted, $seatsMaterial){
        parent::__construct($license,$driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatsMaterial = $seatsMaterial;
    }
    public function setPassenger($passenger){
    if($passenger == 6){
        $this->passenger = $passenger;
    }else{
        echo "<br>Necesitas asignar seis pasajeros";
    }
    }
}
?>

File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
  Map<String, Map<String,Integer>> typeCarAccepted;
  ArrayList<String> seatsMaterial;
  public UberVan(String licence, Drver driver, Map<String, Map<String,Integer>> typeCarAccepted,ArrayList<String> seatsMaterial){
     super(licence, driver);
     this.typeCarAccepted = typeCarAccepted;
     this.seatsMaterial = seatsMaterial;
   }
  public UberVan(String licence, Drver driver){
     super(licence, driver);
   }
   @Override
   public void setPassenger(Integer passenger) {
     if(passenger == 6) {
              this.passenger = passenger;
       }else{
              System.out.println("Necesitas asignar seis pasajeros");
       }
   }
  

}
